src/utils/math.helper.ts

```diff
@@ -0,0 +1,4 @@
+export function isNumber(value: unknown) {
+  if (typeof value === "number") return true
+  return false
+}
\ No newline at end of file
```

Comment:
It's a good practice to end files with a newline character. This can prevent potential issues with certain version control systems or text editors and adheres to the POSIX standard.